GameDefinition/Content/ItemKind.hs

```diff
@@ -540,7 +540,7 @@ potion2 = potionTemplate
                , toOrganGood "hasted" (20 + 1 `d` 5)
                , OnSmash (Explode "pheromone")
                , OnSmash (Explode "haste spray") ]
-  -- , idesc    = ""
+  , idesc    = "The liquid fizzes with energy."
```

Comment:
Splendid. The "energy" rhymes with the sexual subtext (in a family-friendly packaging) of `ELabel "of Attraction"` and of `pheromone`.